lab-davide/gulpfile.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const gulp =  require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+const nodemon = require('gulp-nodemon');
+
+const paths = ['*./js', 'lib/*.js', 'model/*.js;', 'test/*.js', 'route/*.js'];
+
+
+gulp.task('eslint', function(){
+  gulp.src(paths)
+  .pipe(eslint())
+  .pipe(eslint.format());
+});
+
+gulp.task('test', function(){
+  return gulp.src('test.js', {read: false})
```

Comment:
This doesn't do anything--it would only test a file called test.js in your root directory. It doesn't do anything with your actual test files in the test directory.